python/tests/core/clipping/test_add_clipping_geometries.py

```diff
@@ -22,15 +22,16 @@
 
 import brayns
 from tests.mock_instance import MockInstance
+from tests.mock_model import MockModel
 
 
 class TestAddClippingGeometries(unittest.TestCase):
 
     def test_add_clipping_geometry(self) -> None:
-        instance = MockInstance(0)
+        instance = MockInstance(MockModel.message)
         plane = brayns.ClipPlane(1, 2, 3, 4)
-        test = brayns.add_clipping_geometry(instance, plane)
-        self.assertEqual(test, 0)
+        test_model = brayns.add_clipping_geometry(instance, plane)
+        self.assertEqual(test_model, MockModel.model)
```

Comment:
In the add_light tests you left it named "model", better to stay coherent.

Reply:
Addressed